lib/project_types/extension/models/specification_handlers/beacon_extension.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+require "base64"
+require "fileutils"
+require "json"
+require_relative "beacon_extension_utils/script_config"
+require_relative "beacon_extension_utils/script_config_repository"
+
+module Extension
+  module Models
+    module SpecificationHandlers
+      class BeaconExtension < Default
+        SCRIPT_FILE = "beacon-extension.js"
```

Comment:
also address Chris' comment to avoid using snake case

Reply:
it is now kebab case